.github/dependabot.yml

```
@@ -16,3 +16,28 @@ updates:
       - dependency-name: "pdfjs-dist"
       - dependency-name: "@types/*"
       - dependency-name: "typescript"
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

```suggestion
      interval: "weekly"
    groups:
      github-actions:
        patterns:
          - "*"
```

add it to every entry with github-actions ecosystem, it's usually more convenient to get a single PR for all the updates

also, consider setting the interval to monthly as GHA updates usually are never high priority

---

Done!